e2e-tests/k8s-e2e-bootstraping.sh

```diff
@@ -21,7 +21,7 @@ setup_kubectl() {
 }
 
 start_minikube() {
-    $E2E_SUDO minikube start --vm-driver="$E2E_MINIKUBE_DRIVER" --bootstrapper=kubeadm --kubernetes-version="$E2E_KUBERNETES_VERSION" --logtostderr
+    $E2E_SUDO minikube start --driver="none" --kubernetes-version="$E2E_KUBERNETES_VERSION" --logtostderr
```

Comment:
This `"none"` here will trigger auto-detection, right? If yes, can we just not send this argument then?\ and remove the `E2E_MINIKUBE_DRIVER` variable?

Reply:
Let me try

Reply:
I'm going to leave it, but set it to the env var, that way I can run the e2e and set my driver to vmware :)

We set the env var in the `.travis` file